.circleci/config.yml

```diff
@@ -129,6 +129,9 @@ jobs:
     steps:
       - checkout
       - install_js
+      - run:
+          name: Tests charts
+          command: yarn test:charts:unit
```

Comment:
Can we add a TODO to signal that we want to remove this once ESM issues are solved? It's weird to have different ways to run the tests.

Reply:
Another option could be to have a `test:unit` on every package and then call `workspace run test:unit` in the root package `test:unit`, that way having a custom config for charts would be a bit less "hacky". 🙈 
WDYT?

Reply:
I prefer to keep the hacky feeling, just such that when reading this code it's obvious that something is wrong and need to be fixed (once ESM is done)